adr/2022-08-decouple-from-ietf.md

```diff
@@ -0,0 +1,82 @@
+  using it to version our releases a while ago. We now use a date based version
+  and even have more than one draft submission per release (the initial release
+  and the patch release).
+* The IETF process is optimized for working on a draft until it's done and then
+  disbanding. In some cases, the RFC may be revisited and revised in the future,
+  but this is rare and generally contains little more than clarifications and
+  reference updates. JSON Schema is not like that. JSON Schema is more like a
+  programming language. When it stops evolving, it will loose it's relevance.
+  When we finish a release of JSON Schema, we don't disband, we start work on
+  the next release.
+* Every one of our releases is expected to be used in production and will be
+  depended on for many years forward. This is not consistent with normal IETF
+  drafts. Even if we don't publicly use the term "draft", we're still using the
+  IETF I-D system in a way that's not intended.
+* Under IETF, JSON Schema fits under the category of "draft". The community has
+  repeatedly told us that they perceive this to meant that JSON Schema
+  "incomplete" and not "not ready for production use". This is the wrong message
+  for us to be sending as all of our releases are intended to be used in
+  production. This ADR doesn't decide whether or not to drop the "draft" from
+  our releases, but decoupling from IETF gives us that option.
+* Several members of the JSON Schema team have had poor interactions with IETF
+  and don't feel that working with them would be productive. This is a
+  relatively minor consideration. If we thought IETF was right for JSON Schema,
+  we could find ways to make those relationships work.
+
+## Considered Options
+
+1. Continue to submit I-Ds, while using our customized process with no intention
+  of pursing standards track RFC status.
+2. Get JSON Schema to a stable place and pursue a standards track RFC with the
+  IETF.
+3. Decouple from IETF completely and come up with our own governance model.
+
+## Decision Outcome
+
+Our decision is to go with Option 3 and decouple from IETF completely and come
+up with our own governance model. The idea of having the come up with our own
+governance model is daunting, but when the alternatives are to continue to abuse
+the I-D process or to conform to the IETF process that doesn't fit our needs,
+going our own way seems the bast option.
+
+### Positive Consequences
+
+* Decoupling from IETF allows us to distance ourselves from the assumptions that
+  people make about JSON Schema because they assume it works like a typical I-D.
+* Decoupling from IETF allows us to customize our governance model to what works
+  best for JSON Schema.
+
+### Negative Consequences
+
+* Choosing our own governance model means we aren't associated with a recognized
+  standard body which could reduce to credibility of JSON Schema in the eyes of
+  some.
```

Comment:
This is where putting an emphasis on our involvement with the OpenJS Foundation / Linux Foundation is important, and why I think that decision belongs in this ADR.  The ADR will be a public statement of intent, and stating that we plan to work within that context is a more re-assuring one than stating that we're abandoning the IETF without a new plan.

Reply:
> stating that we're abandoning the IETF without a new plan.

I felt uncomfortable writing this up saying we're sure we don't want IETF without being able to say precisely (or even vaguely) what we do want. Maybe that means it's too early for this ADR?

> This is where putting an emphasis on our involvement with the OpenJS Foundation / Linux Foundation is important

I disagree. While OpenJS does provide credibility in some ways, I was referring to the credibility of being a standard recognized by recognized standards body (IETF, W3C, IEEE, etc). OpenJS is not standards body. I can reword to make that perspective more clear. We are already part of OpenJS and nothing we are deciding here changes that.

Reply:
> I felt uncomfortable writing this up saying we're sure we don't want IETF without being able to say precisely (or even vaguely) what we do want. Maybe that means it's too early for this ADR?

Maybe - if the purpose of the discussion here is to sort that out, then it's fine.  But yeah, I think having at least a vague statement of what we're doing instead is important, and if we're not at the point of figuring that out, we should focus on that.